### PR TITLE
feat: env variables for configuring vp9

### PR DESCRIPTION
**Pull request by prayagsingh**

### CODE REVIEW

**Comment by prayagsingh** [comment]
@saghul Please review it.

please don't merge it as of now because when VP9 is enabled then Screenshare is causing a crash if the chrome/chromium version is 92. It's working fine on chromium v93. I have tested it today only.
 OR 
we can add a note about updating the chrome/Brave/Edge browsers to version 93 if using VP9.

File: web/rootfs/defaults/settings-config.js [hunk 1]
```diff
@@ -313,8 +313,6 @@ config.deploymentInfo.userRegion = '{{ $DEPLOYMENTINFO_USERREGION }}';
 
 
 // Testing
-//
```
Comment (saghul) [inline web/rootfs/defaults/settings-config.js:316]: LEave these please, I've used that style of comments to designate a "section"

**Review by saghul — APPROVED** [review]
LGTM with a minor nit. Since VP9 is not enabled by default this is safe to land.

File: web/rootfs/defaults/settings-config.js (hunk 1 above)
Comment (prayagsingh) [reply web/rootfs/defaults/settings-config.js:316]: ok

**Review by saghul — APPROVED** [review]
LGTM

**Pull request merged**